spec/unit/runner/variant-reassignment.spec.js

```diff
@@ -367,4 +367,32 @@ describe('Variant reassignment', () => {
       expect(testFunction.callCount).to.equal(0)
     })
   })
+
+  describe('resolve product type references', () => {
+    it('should resolve product type references', async () => {
+      const productServiceMock = new ProductManager(utils.logger, {})
+      sinon.stub(productServiceMock, 'fetchProductsFromProductDrafts')
+        .resolves(null)
+      const variantReassignments = new VariantReassignment(null, logger)
+      sinon.stub(variantReassignments, '_processUnfinishedTransactions')
+        .resolves(null)
+      const testFunction = sinon.stub(variantReassignments, '_selectProductDraftsForReassignment')
+        .returns([])
+      variantReassignments.productService = productServiceMock
+
+      const productTypeKey = 'productTypeKey'
```

Comment:
We resolve productType reference using `productType.name` but here we use `productTypeKey`

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/57/commits/3d17ad118cdc9d4aad943f529101d52d6f3d57d8